Tests/Command/RemoveThumbsCommandTest.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Tests\Command;
+
+use Sonata\MediaBundle\Command\RemoveThumbsCommand;
+use Sonata\MediaBundle\Model\MediaManagerInterface;
+use Sonata\MediaBundle\Provider\Pool;
+use Sonata\MediaBundle\Tests\Entity\Media;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\Filesystem\Tests\FilesystemTestCase;
+
+/**
+ * @author Anton Dyshkant <vyshkant@gmail.com>
+ *
+ * @requires function Symfony\Component\Console\Tester\CommandTester::setInputs
+ */
+final class RemoveThumbsCommandTest extends FilesystemTestCase
+{
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject|ContainerInterface
+     */
+    private $container;
+
+    /**
+     * @var Application
+     */
+    private $application;
+
+    /**
+     * @var RemoveThumbsCommand
+     */
+    private $command;
+
+    /**
+     * @var CommandTester
+     */
+    private $tester;
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject|Pool
+     */
+    private $pool;
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject|MediaManagerInterface
+     */
+    private $mediaManager;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $this->container = $this->getMockBuilder('Symfony\Component\DependencyInjection\ContainerInterface')->getMock();
+
+        $this->command = new RemoveThumbsCommand();
+        $this->command->setContainer($this->container);
+
+        $this->application = new Application();
+        $this->application->add($this->command);
+
+        $this->tester = new CommandTester($this->application->find('sonata:media:remove-thumbnails'));
+
+        $this->pool = $pool = $this->getMockBuilder('Sonata\MediaBundle\Provider\Pool')->disableOriginalConstructor()->getMock();
+
+        $this->mediaManager = $mediaManager = $this->getMockBuilder('Sonata\MediaBundle\Model\MediaManagerInterface')->getMock();
```

Comment:
Please line-break every line that has more than 80 chars https://dev.to/greg0ire/why-i-keep-pestering-everyone-about-long-lines/

Reply:
BTW, using `MediaManagerInterface::class` would help a lot.

Reply:
And no @jlamur, the max is not 80 but 120.

http://www.php-fig.org/psr/psr-2/

Reply:
The rule I personally apply is: line-break every line that has more than 120 chars to a few lines that have less than 80 chars.